src/lib/utilities/route-for-api.ts

```diff
@@ -1,48 +1,67 @@
+import urlcat from 'urlcat';
+
 let base = (import.meta.env?.VITE_API as string) ?? process.env.VITE_API;
-if (base.endsWith('/')) base = base.slice(0, -1);
+base = `${base}/api/v1/`;
```

Comment:
could you add few tests for relative paths when `VITE_API` is set to empty string or `/`